packages/repository/src/__tests__/fixtures/repositories/address.repository.ts

```diff
@@ -29,7 +29,7 @@ export class AddressRepository extends DefaultCrudRepository<
     customerRepositoryGetter: Getter<CustomerRepository>,
   ) {
     super(Address, db);
-    this.customer = this._createBelongsToAccessorFor(
+    this.customer = this.createBelongsToAccessorFor(
```

Comment:
Because of this change, there's no longer coverage for `_createBelongsToAccesorFor`: https://coveralls.io/builds/24455533/source?filename=packages/repository/src/repositories/legacy-juggler-bridge.ts#L259. Should I revert it?

Reply:
It's deprecated, so not sure what the best course of action should be. For now, we could duplicate the tests for coverage of both until we remove it. @bajtos thoughts?

Reply:
I reverted it so the function is still covered, but I can open another PR if we want to change it to the non-deprecated version.